workers/data_refinery_workers/processors/smashing_utils.py

```diff
@@ -387,6 +397,20 @@ def process_frames_for_key(key: str,
                     job_context['rnaseq_matrix'] = rnaseq_chunk_frame
 
                 del rnaseq_chunk_frame
+                # start using dask to save ram at scale
+                if index > INDEX_DASK_START and type(job_context['rnaseq_matrix']) is pd.DataFrame:
+                    job_context['rnaseq_matrix'] = dd.from_pandas(job_context.pop('rnaseq_matrix'),
+                                                                      npartitions=MULTIPROCESSING_WORKER_COUNT)
+
+            log_state("end merging chunk of frames",
+                      job_context["job"],
+                      start_frame_chunk)
```

Comment:
Can we get a counter on this and the `merging chunk of frames` message as well? Otherwise we're just gonna have 250 of these with no indication of progress.

Reply:
well it would be after the processing index line